object_intern.go

```diff
@@ -168,55 +211,116 @@ func (oi *ObjectIntern) AddOrGet(obj []byte, safe bool) (uintptr, error) {
 // If the object is found in the store its reference count is increased by 1.
 // If the object is added to the store its reference count is set to 1.
 func (oi *ObjectIntern) AddOrGetString(obj []byte, safe bool) (string, error) {
```

Comment:
Wouldn't it be nicer if `AddOrGetString()` internally uses `AddOrGet()`

Reply:
Maybe, but they might diverge further in the future. My other concern is that none of those methods are simple enough to be inlined.